dashboard/app/models/script.rb

```diff
@@ -199,13 +199,16 @@ def self.stage_extras_script_ids
   # @param [User] user
   # @return [Script[]]
   def self.valid_scripts(user)
-    with_hidden = user.permission?(UserPermission::HIDDEN_SCRIPT_ACCESS)
+    user_experiments_enabled = user.any_experiments_enabled?
```

Comment:
it would be better to use `Course.has_any_course_experiments?(user)` here, both for code reuse and because it only checks for experiments which are course experiments.